includes/admin/post-types/meta-boxes/class.llms.meta.box.order.submit.php

```diff
@@ -136,6 +137,13 @@ public function save( $post_id ) {
 			'_llms_date_access_expires',
 		);
 
+		// Do not save recurring payments related dates if order's gateway do not support recurring payments modification.
+		if ( ! $order->supports_modify_recurring_payments() ) {
+			$editable_dates = array(
+				'_llms_date_access_expires',
+			);
+		}
+
```

Comment:
I'd rather see the array start as the dates that are always editable (access_expires) and then add the conditional ones later if the order supports recurring payments in favor of overwriting / redeclaring the array with only the expires date when the order cannot modify recurring payments.

Reply:
Ok, makes sense.